packages/evasive-transform/src/transform-comment.js

```diff
@@ -46,3 +46,44 @@ export function transformComment(node, unmapLoc) {
     unmapLoc(node.loc);
   }
 }
+
+/**
+ * Inspects a comment for a hint that it must be preserved by a transform.
+ *
+ * @param {string} comment
+ */
+const markedForPreservation = comment => {
+  if (comment.startsWith('!')) {
+    return true;
+  }
+  if (comment.startsWith('*')) {
+    // Detect jsdoc style @preserve, @copyright, @license, @cc_on (IE
+    // cconditional comments)
+    return /(?:^|\n)\s*\*?\s*@(?:preserve|copyright|license|cc_on)\b/.test(
+      comment,
+    );
+  }
+  return false;
+};
+
+/**
+ * Rewrites comments by replacing their interior with an ellipsis, deleting all
```

Comment:
I expect most readers will think ellipsis=`…`

Consider,
> Rewrites comments by blanking their interior, deleting all non-newlines before the last line.

Reply:
Thank you for the catch. I removed the ellipsis and did not update the comment.